src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days >= 3 && days <= 6) {
+    sum = 40 * days - 20;
+
+    return sum;
```

Comment:
You don't need the `sum` variable here. You can return the result of the calculation directly, which simplifies the code and adheres to checklist item 4.